### ISSUE TITLE
Compatibility with Blackwell GPU (RTX50 series)

### ISSUE THREAD

**Issue by boxiao-wv** [Pointcept/Pointcept#465]
I wonder whether anyone has succesfully run PT-V3 with the newest RTX50 series graphic card? 

With pytorch==2.7.0 and cuda==12.8, I can run inference but training always fails at the `loss.backward()` step. Any inputs are helpful!

**Comment by jj-curtis** [comment]
Did you ever find a solution to this? Thanks.
